### PR TITLE
milestone: Portfolio setup2;

### PR DESCRIPTION
**Pull request by kelwilson**
### Add a section;

- Add an H1 tag and a picture image
- Add images
- Add list tags
- Add div tags
- Add background-images
- Add some texts to describe the projects 

### Add an about section ;

- Add more details about the project
- Add an H3 and a list of languages 
- Add an H3 and a list of frameworks
- Add an H3 and a list of skills
- Add some more background images

### Add styles ;

- Add styles to all the list 
- Add styles to all the images
- Add styles to all the buttons to make them responsive
- Update some semantic tags and made a few changes to the styles
00f6b750637041fbb59c4f8e785d3464
d1f3ee7089444b1191f6d7a8713a97f9
d415c459a1734a759bd86ffc76e7d073

**please thanks for the review, I repeated the first week and it had a great toll on me coz of fatigue and lack of sleep I gave my all into these projects, I'm always willing to learn, so please be free to criticize my work, and help me get better by telling me about my mistakes and teaching me how to do better at my code and all, please my time is almost up for submission and I don't want to repeat the week by moving to the next cohort. thanks and I would be waiting for your critics and also your encouragement. sorry for the message though I know its unprofessional **

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,5 +55,156 @@ <h2 class="start-btn">Start collaboration</h2>
...
-  </body>
+    <section class="project-header">
+      <div class="header-div">
+        <h1 class="header-2">Projects</h1>
+
+        <div class="project-img">
+          <img src="images/first.png" class="image-1" alt="" />
+        </div>
+      </div>
+      <div class="project-details">
```
Comment (danielufeli) [inline index.html:68]: - [x] Nice work you have done with the projects section, kindly add some padding between the image and the title to give it some space, so it aligns with what is in the Figma Template.
- [x] Kindly align the see this project `button to the left and give it some padding so it matchess what we have in the Figma Template.
- [x] Kindly change the `font` and `font-size` of the text in the tags so it aligns with what is on the Figma Template see below!

## Figma Template
card-n-1

## Your Design
card-n-2

File: index.html [hunk 2]
```diff
@@ -54,5 +55,156 @@ <h2 class="start-btn">Start collaboration</h2>
...
+        </ul>
+      </div>
+      <div class="more-info">
+        <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+        
+      </div>
+    </section>
+    <section class="main-info">
+      <div class="main-info-div">
+        <h1 class="header-3">About</h1>
```
Comment (danielufeli) [inline index.html:187]: - [x] Nice working on the About us section kindly add some `line-space` in between the paragraph text so it aligns with the Figma Template.
- [x] Kindly add padding to the list in the `.more-info` cards and make the text to be bolder, so it aligns with the Figma Template.

## Figma Template
abt-1

## Your Design
abt-2

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @kelwilson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Highlights
Nice commits messages✔️
Descriptive PR ✔️
Clean Design ✔️
There is no linter error is present. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kelwilson) [reply index.html:68]: thanks, i hope I met the requirements now, I'm mentally and physically drained, worked for two days straight without sleep just to complete these projects and meet up with the deadline. thanks once more

**Review by smunozmo — APPROVED** [review]
Hi @kelwilson, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**